src/Paysera/Bundle/CodeGeneratorBundle/Entity/Definition/FilterTypeDefinition.php

```diff
@@ -7,17 +7,43 @@ class FilterTypeDefinition extends TypeDefinition
     /**
      * @var bool
      */
+    private $baseFilter;
+
+    /**
+     * @var bool
+     */
     private $extendsBaseFilter;
 
     public function __construct()
     {
         parent::__construct();
+
+        $this->baseFilter = false;
         $this->extendsBaseFilter = false;
     }
 
     /**
      * @return bool
      */
+    public function isBaseFilter()
+    {
+        return $this->baseFilter;
+    }
+
+    /**
+     * @param bool $baseFilter
+     *
+     * @return $this
+     */
+    public function setBaseFilter($baseFilter)
```

Comment:
Either param type should be strict, or param type of setExtendedBaseFilter shouldn't. Also `declare(strict_types=1);` is missing